package.json

```diff
@@ -25,33 +29,33 @@
         ]
     },
     "dependencies": {
+        "@eigenspace/jest-testing-utils": "0.0.1",
         "enzyme": "3.8.0",
         "enzyme-to-json": "3.3.5",
         "jest": "24.7.1",
         "jest-image-snapshot": "2.8.1",
+        "jest-styled-components": "6.3.1",
         "react": "16.8.1",
         "react-dom": "16.8.1",
-        "react-test-renderer": "16.8.1"
+        "react-test-renderer": "16.8.1",
+        "styled-components": "4.1.3"
```

Comment:
Do we really want to require styled components as dependency, not peer dependency?

Reply:
I think, yes. It remains only to agree on how we organize package.json for dependencies. This is the problem of all our projects.